.github/workflows/index.yml

```diff
@@ -0,0 +1,27 @@
+---
+name: Indexing
+on: workflow_dispatch
+jobs:
+  build-and-index:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Yarn Install
+        uses: bahmutov/npm-install@v1
+      - name: Build
+        run: |
+          yarn build
+        env:
+          PREFIX_PATHS: true # equivalent to --prefix-paths flag for 'gatsby build'
+          REPO_GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          REPO_OWNER: ${{ github.repository_owner }}
+          REPO_NAME: ${{ github.event.repository.name }}
+          REPO_BRANCH: ${{ github.ref_name }}
+          GATSBY_ALGOLIA_APPLICATION_ID: ${{ secrets.AIO_ALGOLIA_APPLICATION_ID }}
+          GATSBY_ALGOLIA_SEARCH_API_KEY: ${{ secrets.AIO_ALGOLIA_SEARCH_API_KEY }}
+          ALGOLIA_WRITE_API_KEY: ${{ secrets.AIO_ALGOLIA_WRITE_API_KEY }}
+          ALGOLIA_INDEXATION_MODE: index
```

Comment:
It would be good to use `console` for testing to avoid over using the API.

Reply:
good point

Reply:
`GATSBY_ALGOLIA_SEARCH_INDEX` is no longer needed. We've also moved to an external product-index map which you can find along with latest indexing instructions here https://github.com/AdobeDocs/search-indices

Reply:
@dmitrymatio, is https://github.com/AdobeDocs/dev-site/blob/main/static/shared/search/index.js still relevant? Or https://github.com/AdobeDocs/search-indices replaces it?